projects/plugins/jetpack/extensions/plugins/ai-assistant-plugin/components/seo-assistant/seo-assistant-wizard.tsx

```diff
@@ -1,21 +1,34 @@
+/**
+ * External dependencies
+ */
+import { useDispatch } from '@wordpress/data';
 import debugFactory from 'debug';
-import './style.scss';
+/**
+ * Internal dependencies
+ */
+import { STORE_NAME } from './store';
```

Comment:
the latest trend here is to always export `store` and import it as someStore:

```js
import { store as seoAssistantStore } from './store';
```
at least, this is how core/editor/data/etc stores are being exported/imported lately